gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/search/lucene/transformer/ApiDocumentTransformer.java

```diff
@@ -157,6 +159,10 @@ public Document transform(io.gravitee.rest.api.model.api.ApiEntity api) {
         return doc;
     }
 
+    private BytesRef toSortedValue(String value) {
+        return new BytesRef(value.replaceAll("[|\\-+!(){}^\"~*?:&\\/]", "").toLowerCase());
```

Comment:
You could extract the regex in a final variable so the pattern compilation occurs only once.

Reply:
👌